cadvisor_test.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2014 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"testing"
+
+	"github.com/google/cadvisor/container"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTcpMetricsAreDisabledByDefault(t *testing.T) {
+	assert.True(t, ignoreMetrics.Has(container.NetworkTcpUsageMetrics))
```

Comment:
This test isn't really doing anything anymore, since the ignoreMetrics flag is never actually added. I think the best way to solve this is by moving the flag declaration to an `init()` function, or an `addFlags()` function that you can call here.

Also, consider adding a `flag.Parse()` and verifying that the assertion still holds.